src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -1130,8 +1130,6 @@ namespace InternalHeatGains {
             if (Zone(Loop).isNominalControlled) { // conditioned zones only
                 if (Zone(Loop).TotOccupants > 0.0) {
                     Zone(Loop).isNominalOccupied = true;
-                    PreDefTableEntry(pdchOaoNomNumOcc1, Zone(Loop).Name, Zone(Loop).TotOccupants);
-                    PreDefTableEntry(pdchOaoNomNumOcc2, Zone(Loop).Name, Zone(Loop).TotOccupants);
```

Comment:
@JasonGlazer This change is causing the order of zones to change in some output table reports, such as ABUPS Time Not Comfortable and Time Setpoint Not Met and a few other sundry reports. Because this was the first place `PreDefTableEntry` was called with zone names, and this loop is only for conditioned zones, the unconditioned zones (like PLENUM-1 in the 5zone files) weren't in the name list yet.  So this is the source of the big table diffs, because some tables have the zone rows in a different order (it ignores the new table because it doesn't match in size). It's not a bad thing, because all of the table now list the zones in the same order. Just noting it here.